packages/react-components/src/components/OMBInfo/OMBInfo.jsx

```diff
@@ -18,10 +18,10 @@ class OMBInfo extends React.Component {
     this.setState({ modalOpen: false });
   };
 
-  modalContents = minutes => (
+  modalContents = (minutes, hideResBurdenSection) => (
     <div>
       <h3 id={`${this.id}-title`}>Privacy Act Statement</h3>
-      {minutes && (
+      {minutes && !hideResBurdenSection && (
```

Comment:
Can this already be accomplished by just not passing in a `resBurden` prop?

Reply:
This is interesting, because unless I scrolled down in the Storybook story, I would not have assumed that this is how that prop works. And I didn't see any other documentation on this component. So while yes, that prop does what we need it to do, it's unclear. Is there a way that we can clarify this and make it more intuitive for our frontend teams?

Reply:
We could add a page for `OMBInfo` on design.va.gov/components that describes use cases, or we could add to the prop description which may be helpful:

![image](https://user-images.githubusercontent.com/2008881/148809058-dce6068e-0929-406e-9a1d-85095ce52a87.png)


Maybe:

> Leave empty to remove any mention of a response burden

Reply:
I think we should probably do both.